backup-pruner.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: create dirs
+  file:
+    name: "{{ item }}"
+    owner: "root"
+    group: "root"
+    mode: "700"
+    state: "directory"
+  with_items:
+    - "/etc/tarsnap"
+    - "/var/cache"
+
+# Note: tarsnapper requires keys to end with a newline
+# which are stripped from ansible variables.
+# tarsnap.key template exists only to add that newline.
+- name: upload keys
+  template:
+    src: tarsnap.key
+    dest: "{{ item.keypath }}"
+    owner: "root"
+    group: "root"
+    mode: "400"
+  no_log: true
+  with_items: "{{ keys_to_upload }}"
+
+- name: create cache dirs
+  file:
+    name: "{{ item.cache }}"
+    owner: "root"
+    group: "root"
+    mode: "700"
+    state: directory
+  no_log: true
+  with_items: "{{ keys_to_upload }}"
+
+# NOTE: Pip doesn't work with tarsnapper
+- name: install tarsnapper
+  command: easy_install "{{ TARSNAPPER_VERSION }}"
```

Comment:
Why do you need to install tarsnapper manually, when the `pmbauer.tarsnap` role from https://github.com/open-craft/deployment-deploy-prune-backups/pull/1/files#diff-976d348c79e886d511b3d19dc93fa69dR8 should already allow to install it?